### PR TITLE
holdingpen: allow passing a redirect_url on save

### PR DESCRIPTION
**Pull request by david-caro**
Signed-off-by: David Caro <david@dcaro.es>

### CODE REVIEW

**Comment by drjova** [comment]
LGTM

**Comment by drjova** [comment]
hmm the tests are failing for a weird reason, checking

**Comment by david-caro** [comment]
I don't know why it fails the tests though.

File: .travis.yml [hunk 1]
```diff
@@ -13,6 +13,7 @@ before_install:
 - export CHROME_BIN=chromium-browser
 - export DISPLAY=:99.0
 - sh -e /etc/init.d/xvfb start
+- npm install
```
Comment (harunurhan) [inline .travis.yml]: I think this is run by default

File: src/app/holdingpen-editor/holdingpen-save-button/holdingpen-save-button.component.ts [hunk 2]
```diff
@@ -99,8 +99,12 @@ export class HoldingpenSaveButtonComponent extends SubscriberComponent implement
         } else {
           const referrer = document.referrer;
```
Comment (harunurhan) [inline src/app/holdingpen-editor/holdingpen-save-button/holdingpen-save-button.component.ts]: can you put this and next line into else block.

File: .travis.yml (hunk 1 above)
Comment (david-caro) [reply .travis.yml]: see this one for example: https://travis-ci.org/inspirehep/record-editor/jobs/395351729

Comment (david-caro) [reply .travis.yml]: here too, it ran it only once: https://travis-ci.org/inspirehep/record-editor/jobs/395390501

Maybe they changed it for newer node?

**Comment by david-caro** [comment]
@drjova @harunurhan now it passes all the tests, last review?

File: .travis.yml (hunk 1 above)
Comment (drjova) [reply .travis.yml]: actually is using https://docs.npmjs.com/cli/ci which expects ```package-lock.json`` I think

Comment (david-caro) [reply .travis.yml]: hmm... so that package-lock.json should be added to the repo?

**Pull request merged**